src/GitHub.App/ViewModels/PullRequestDirectoryViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GitHub.ViewModels
+{
+    public class PullRequestDirectoryViewModel : IPullRequestDirectoryViewModel
```

Comment:
Actually, this is only used internally in the `PullRequestDetailViewModel`, so it probably shouldn't be exposed publicly, it's an implementation detail. We can just have this class in the `PullRequestDetailViewModel` as a helper class. I would definitely still not call it a ViewModel - the only thing it does is represent data, it has no logic for a view to interact with data, it's just a handy model (or in this case, a handy helper class)